src/SignupRouter.js

```diff
@@ -19,18 +19,20 @@ import { getOriginalScreenHeight } from './lib/utils/Orientation'
 
 const initialRouteName = isMobileSafari && isWebApp ? 'IOSWebAppSignIn' : 'Auth'
 const AuthType = Config.torusEnabled ? AuthTorus : Auth
-const router = createSwitchNavigator(
-  {
-    Auth: AuthType,
-    Signup,
-    InvalidW3TokenError,
-    SigninInfo,
-    IOSWebAppSignIn,
-  },
-  {
-    initialRouteName,
-  }
-)
+
```

Comment:
check also routes in Dashboard pls

Reply:
> > > Couldn't find where route route to "Good To Know" is removed ?
> > 
> > 
> > @serdiukov-o-nordwhale
> > Changes in the SignUpState.js file.
> > See this commit - [bb2165c](https://github.com/GoodDollar/GoodDAPP/commit/bb2165c0a51c10dcbb7a60bf35d1f9a578e49225)
> 
> How about MagicLinkInfo in src/components/dashboard/Dashboard.js ?
> it contains {'GOOD TO KNOW'} in title i think @sirty mean this one

This described in #1376 task - so I will do it there